src/views/collection-view/index.tsx

```diff
@@ -20,6 +21,7 @@ function CollectionView({
   layoutProps,
   product,
 }: CollectionPageProps): React.ReactElement {
+  useOptInAnalyticsTracking('learn')
```

Comment:
The analytics hook is called only on the views that handle opt-out UI. We could eventually lift this up, but this feels appropriate for now.